pages/manager_page.py

```diff
@@ -0,0 +1,70 @@
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.ui import WebDriverWait
+
+from pages.locators import AllLocators
+
+
+class ManagerPage:
+    def __init__(self, browser):
+        self.browser = browser
+        self.url = "https://www.globalsqa.com/angularJs-protractor/BankingProject/#/manager"
+
+    def open(self):
+        self.browser.get(self.url)
+        element_present = EC.visibility_of_element_located((By.XPATH, "//button[@ng-class='btnClass1']"))
+        WebDriverWait(self.browser, 5).until(element_present)
+
+    def open_add_customers_form(self):
+        self.browser.find_element(*AllLocators.ADD_CUSTOMER_OPEN_FORM_BUTTON).click()
+        element_present = EC.visibility_of_element_located((By.XPATH, "//input[@ng-model='fName']"))
+        WebDriverWait(self.browser, 5).until(element_present)
+
+    def open_customers_data(self):
+        self.browser.find_element(*AllLocators.CUSTOMER_BUTTON).click()
+        element_present = EC.visibility_of_element_located((By.TAG_NAME, "table"))
+        WebDriverWait(self.browser, 5).until(element_present)
+
+    def set_first_name(self, name):
+        name_input = self.browser.find_element(*AllLocators.FIRST_NAME_FIELD)
+        name_input.send_keys(name)
```

Comment:
Здесь и далее. Если не используем переменную более одного раза:
```suggestion
    def set_first_name(self, name):
        self.browser.find_element(*AllLocators.FIRST_NAME_FIELD).send_keys(name)
```